arcgis-ios-sdk-samples/Display information/Query features with Arcade expression/QueryFeaturesArcadeExpressionViewController.swift

```diff
@@ -0,0 +1,107 @@
+// Copyright 2021 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class QueryFeaturesArcadeExpressionViewController: UIViewController {
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            mapView.map = makeMap()
+            // Set the touch delegate.
+            mapView.touchDelegate = self
+        }
+    }
+    
+    /// The Arcade expression evaluation operation.
+    var evaluateOperation: AGSCancelable?
+    
+    /// Make and load a map.
+    func makeMap() -> AGSMap {
+        // Create a portal item with the portal and item ID.
+        let portalItem = AGSPortalItem(portal: .arcGISOnline(withLoginRequired: false), itemID: "539d93de54c7422f88f69bfac2aebf7d")
+        // Make a map with the portal item.
+        let map = AGSMap(item: portalItem)
+        return map
+    }
+    
+    /// Evaluate the Arcade expression for the selected feature at the map point.
+    func evaluateArcadeInCallout(for feature: AGSArcGISFeature, at mapPoint: AGSPoint) {
+        // Instantiate a string containing the Arcade expression.
+        let expressionValue =
+        """
+        var crimes = FeatureSetByName($map, 'Crime in the last 60 days');
+        return Count(Intersects($feature, crimes));
+        """
+        // Create an Arcade expression using the string.
+        let expression = AGSArcadeExpression(expression: expressionValue)
+        // Create an Arcade evaluator with the Arcade expression and an Arcade profile.
+        let evaluator = AGSArcadeEvaluator(expression: expression, profile: .formCalculation)
+        guard let map = mapView.map else { return }
+        let profileVariables = ["$feature": feature, "$map": map]
+        evaluateOperation?.cancel()
+        // Show progress hud.
+        UIApplication.shared.showProgressHUD(message: "Evaluating")
+        // Get the Arcade evaluation result given the previously set profile variables.
+        evaluateOperation = evaluator.evaluate(withProfileVariables: profileVariables) { [weak self] result, error in
+            // Dismiss progress hud.
+            UIApplication.shared.hideProgressHUD()
```

Comment:
```suggestion
        // Get the Arcade evaluation result given the previously set profile variables.
        evaluateOperation = evaluator.evaluate(withProfileVariables: profileVariables) { [weak self] result, error in
```
If we gonna proceed with the buggy evaluate operation, then we need to not use the HUD so it doesn't block user interaction, and when the evaluation hangs, nothing is showing in UI level.